gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -174,6 +174,10 @@
   public static final String CONVERTER_STRING_SPLITTER_DELIMITER = "converter.string.splitter.delimiter";
   public static final String CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR = "converter.csv.to.json.enclosedchar";
   public static final String DEFAULT_CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR = "\0";
+  public static final String CONVERTER_AVRO_NULLIFY_FIELDS_ENABLED = "converter.avro.nullify.fields.enabled";
```

Comment:
Please move these config options to the class where they are used: JsonIntermediateToAvroConverter